docs/user_guide/3_getting_started.md

```diff
@@ -178,11 +178,194 @@ When running the one-liner, along with the icon that starts the Grasper, you wil
   ========================   =============       ================    =================================
   ```
 
-## Robot Monitor
+## Graphical User Interface
+
+The majority of functionality is provided by the software Application Programmer Interface (API). However, a few simple functions are provided in the Graphical User Interface (GUI) to test the hand, validate that it is working correctly, and adjust some of its settings.
+
+### Starting the interface
+You may open the Graphical User Interface to try out some functions of the hand. From the Docker terminal, type:
+```bash
+$ rqt
+```
+
+  This interface contains a number of plugins for interacting with the EtherCAT hand. Most of them are available from the **Plugins → Shadow Robot** menu.
+
+### Robot Monitor
+We can check that everything on the robot is working correctly using the Diagnostic Viewer.
+
+  **Plugins → Robot Tools → Diagnotic Viewer**
+
+  ![robot_monitor](../img/robot_monitor.png)
+
+
+This brings up a dialog box containing a tree of all parts of the robot. All parts should be marked with a green tick.
+
+You can examine one motor in detail by double-clicking on it. This brings up the Motor Monitor dialog. This window can be used to check the status of a motor, or debug any problems.
+
+![monitor_single_motor](../img/monitor_single_motor.png)
+
+The following table has some more information on what each of these fields mean.
+
+```eval_rst
+============================   ==============================================================================================
+Item                           Description
+============================   ==============================================================================================
+Full Name
+Component
+Hardware ID
+Level
+Message                        Any error or status messages
+Motor ID                       This is the motor number. Range [0..19]
+Motor ID in message            For debugging only
+Strain Gauge Left / Right      These are the ADC readings from the two gauges
+Executed Effort
+Motor Flags                    See motor flags table below
+Measured current               Current flowing through the motor (Amps)
+Measured Voltage               The motor power supply voltage. Not the voltage at the motor
+Temperature                    The temperature measured near the motor. The actual motor winding temperature will be higher than this. (ºC)
+Number of CAN messages         Received messages should be twice the transmitted messages
+Force control P, I, D terms    These are the PID terms from inside the motor's torque controller. They may be useful for debugging if plotted.
+Force control F, P, I, D,
```

Comment:
This field appear in readthedocs as two rows, then one doesn't have description. We should also add description to the first fields as it looks weird that their description is empty.